prow/github/fakegithub/fakegithub.go

```diff
@@ -105,6 +105,18 @@ func (f *FakeClient) DeleteComment(owner, repo string, ID int) error {
 	return fmt.Errorf("could not find issue comment %d", ID)
 }
 
+func (f *FakeClient) DeleteStaleComments(org, repo string, number int, isStale func(github.IssueComment) bool) error {
+	comments, _ := f.ListIssueComments(org, repo, number)
```

Comment:
The mungers have this abstracted so we only list once, right? Do we intend for many plugins to have this functionality? Should we attempt to build a similar abstraction here?

Reply:
Thats correct, and something similar will probably be necessary for Prow, but lets wait until there is another plugin that requires stale comment deletion before implementing that.